pkg/arvo/sys/zuse.hoon

```diff
@@ -5592,89 +5676,123 @@
       ?>  ?=({$o *} jon)
       ((ou-raw wer) p.jon)
     ::                                                  ::  ++ou-raw:dejs:format
+    :: Takes a list of key/handler pairs, and returns a list of (unit
+    :: extracted-value)s, of the same length in the same order.
     ++  ou-raw                                          ::  object of units
       |*  wer/(pole {cord fist})
       |=  jom/(map @t json)
       ?-    wer                                         :: mint-vain on empty
-          :: {{key/@t wit/*} t/*}
-          {{key/@t *} t/*}
-        =>  .(wer [[* wit] *]=wer)
+          {{key/@t wit/*} t/*}
         =/  ten  ~|(key+key.wer (wit.wer (~(get by jom) key.wer)))
         ?~(t.wer ten [ten ((ou-raw t.wer) jom)])
       ==
     ::                                                  ::  ++om:dejs:format
+    :: just turn the json map into a map of the same keys to the decoded
+    :: values
     ++  om                                              ::  object as map
       |*  wit/fist
       |=  jon/json
       ?>  ?=({$o *} jon)
       (~(run by p.jon) wit)
     ::                                                  ::  ++op:dejs:format
+    :: parses all keys of the obj with `fel`, decodes all values with
+    :: `wit`, and returns a map from parsed keys to decoded values
     ++  op                                              ::  parse keys of map
       |*  {fel/rule wit/fist}
       |=  jon/json  ^-  (map _(wonk *fel) _*wit)
+      :: first decode all the values
       =/  jom  ((om wit) jon)
+      :: tree -> list of key/val pairs -> parse all keys -> back to map
       %-  malt
       %+  turn  ~(tap by jom)
       |*  {a/cord b/*}
+      :: does this replacement do anything? doesn't the line above
+      :: already put the a and b faces on it?
       =>  .(+< [a b]=+<)
```

Comment:
Good q! This might be an old hack to work around wet gate deficiencies that have since been fixed. It used to struggle with putting faces on things.